--- crossterm-0.17.4/Cargo.toml	2020-05-02T06:10:31+00:00
+++ crossterm-0.17.4/Cargo.toml	2020-05-10T12:37:51.042747+00:00
@@ -71,9 +71,3 @@
 [target."cfg(unix)".dependencies.signal-hook]
 version = "0.1.13"
 features = ["mio-support"]
-[target."cfg(windows)".dependencies.crossterm_winapi]
-version = "0.6.1"
-
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.8"
-features = ["winuser"]
