--- crossterm-0.16.0/Cargo.toml	1970-01-01T00:00:00+00:00
+++ crossterm-0.16.0/Cargo.toml	2020-03-19T00:17:11.172520+00:00
@@ -71,9 +71,3 @@
 [target."cfg(unix)".dependencies.signal-hook]
 version = "0.1.13"
 features = ["mio-support"]
-[target."cfg(windows)".dependencies.crossterm_winapi]
-version = "0.6.1"
-
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.8"
-features = ["winuser"]
