--- crossterm-0.20.0/Cargo.toml	1970-01-01T00:00:00+00:00
+++ crossterm-0.20.0/Cargo.toml	2021-09-03T11:59:21.181532+00:00
@@ -81,9 +81,3 @@
 [target."cfg(unix)".dependencies.signal-hook-mio]
 version = "0.2.1"
 features = ["support-v0_7"]
-[target."cfg(windows)".dependencies.crossterm_winapi]
-version = "0.8"
-
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.9"
-features = ["winuser"]
