--- crossterm-0.17.5/Cargo.toml	2020-05-23T11:31:26+00:00
+++ crossterm-0.17.5/Cargo.toml	2020-05-23T18:16:32.069008+00:00
@@ -71,9 +71,3 @@
 [target."cfg(unix)".dependencies.signal-hook]
 version = "0.1.15"
 features = ["mio-support"]
-[target."cfg(windows)".dependencies.crossterm_winapi]
-version = "0.6.1"
-
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.8"
-features = ["winuser"]
