--- crossterm-0.22.1/Cargo.toml	1970-01-01T00:00:01+00:00
+++ crossterm-0.22.1/Cargo.toml	2022-02-13T11:47:33.053433+00:00
@@ -84,9 +84,3 @@
 [target."cfg(unix)".dependencies.signal-hook-mio]
 version = "0.2.1"
 features = ["support-v0_7"]
-[target."cfg(windows)".dependencies.crossterm_winapi]
-version = "0.9"
-
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.9"
-features = ["winuser"]
