--- crossterm-0.19.0/Cargo.toml	2020-12-28T10:05:26+00:00
+++ crossterm-0.19.0/Cargo.toml	2021-03-21T14:46:54.683397+00:00
@@ -81,9 +81,3 @@
 [target."cfg(unix)".dependencies.signal-hook]
 version = "0.1.15"
 features = ["mio-0_7-support"]
-[target."cfg(windows)".dependencies.crossterm_winapi]
-version = "0.7.0"
-
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.9"
-features = ["winuser"]
